apps/allocations/app/components/App/Accounts.js

```diff
@@ -51,7 +46,6 @@ const Accounts = ({
 Accounts.propTypes = {
   // TODO: Create account shape
   accounts: PropTypes.arrayOf(PropTypes.object).isRequired,
-  onNewAccount: PropTypes.func.isRequired,
```

Comment:
All changes on this file seem out of scope of what the PR title says, might we consider updating the PR title and/or comment to explain the full set of changes going here?

Reply:
I hadn't even written a description of the changes, sorry. Please check out the updated description and let me know if it makes more sense.

Reply:
Thanks for the clarification!

Reply:
Thanks for the clarification!